### ISSUE TITLE
Start debugging button lacks `ariaLabel`

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#181943]
In our accessibility user study testing, a user found that the start debugging button lacked a label. We should fix this even though the `title` property does get read by some screen readers currently - such as VoiceOver.
